projects/js-packages/publicize-components/src/components/manage-connections-modal/polling-notice.tsx

```diff
@@ -0,0 +1,35 @@
+import { Notice, Spinner } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { store as socialStore } from '../../social-store';
+import styles from './style.module.scss';
+
+/**
+ * Polling notice component
+ *
+ * @return React element
+ */
+export function PollingNotice() {
+	const keyringRequest = useSelect( select => select( socialStore ).getKeyringRequest(), [] );
+	const { abortPollingForLastKeyringResult } = useDispatch( socialStore );
+
+	if ( ! keyringRequest?.polling ) {
+		return null;
+	}
+
+	return (
+		<div className={ styles[ 'polling-notice-wrapper' ] }>
+			<Notice
+				className={ styles[ 'polling-notice' ] }
+				status="success"
+				onRemove={ abortPollingForLastKeyringResult }
+			>
+				<Spinner />
+				{ __(
+					'Refreshing the available connections, please wait…',
```

Comment:
I wonder if we could come up with something better here, it sounds a bit confusing to me because we already see connections on the page, it's hard to say what are we referring here. Maybe `Getting your Facebook accounts to connect, please wait..` or something similar?

Reply:
Yeah, that sounds better.